.gitlab/hpsf-gitlab-ci.yml

```diff
@@ -2,19 +2,37 @@ NVIDIA-GH200:
   stage: test
   tags: [nvidia-gh200]
   image: masterleinad/kokkos-nvcc:12.6.1
+  rules:
+    - if: $CI_PIPELINE_SOURCE == "schedule"
+      variables:
+        CDASH_MODEL: "Nightly"
```

Comment:
Where would one define that schedule?  In the GitLab API?

Why would we have the same job setup both for PRs and as a nightly?

Reply:
> Where would one define that schedule? In the GitLab API?

Yes, it's defined in the GitLab API, see https://gitlab.spack.io/kokkos/kokkos/-/pipeline_schedules.

> Why would we have the same job setup both for PRs and as a nightly?

It was easiest to use the same configuration and I find it still useful in case we merged multiple pull requests that have an indirect dependency that would not be detected in pull request testing. Of course, we could change a bunch of things like building in `Debug` mode or with RDC. Ultimately, I want to move forward here so that others can build on top of the `CTestRun.cmake` file that is demonstrated to work for nightly tests as well here.

Reply:
I don't think it is worth the added complexity and I would rather have had different configurations to increase coverage but I am fine merging.